Add default value for APT security repo suite

gbp opened the pull request:
Add fallback default based on distribution version to allow unsupported distributions to install software when explicit security suite config is not available.

Was attempting to install Alaveteli on Trixie but without this change the apt sources are invalid.

gbp committed:
Add default value for APT security repo suite

Add fallback default based on distribution version to allow unsupported
distributions to install software when explicit security suite config is
not available.

sagepe reviewed (changes requested):
Going to add a further commit

sagepe committed:
Remove Buster and add Trixie

This also removes logic around the name of the security suite as with
the end of support for Buster this is now standardised again.
